server/upload.py

```diff
@@ -180,7 +180,7 @@ def handleCsv(dataset, prereviewFolder, user, csvFile):
                     filename)
                 continue
             else:
-                imageItem = imageItems.next()
+                imageItem = next(imageItems)
```

Comment:
This change gives me an error:

      File "/home/vagrant/girder/plugins/isic_archive/server/upload.py", line 183, in handleCsv
        imageItem = next(imageItems)
    TypeError: MongoProxy object is not an iterator

Perhaps Girder's `MongoProxy` doesn't properly implement the iterator methods? https://docs.python.org/2/library/stdtypes.html#iterator-types